wagtail/contrib/wagtailfrontendcache/backends.py

```diff
@@ -91,20 +97,20 @@ def purge(self, url):
                 if response.status_code != 200:
                     response.raise_for_status()
                 else:
-                    logger.error("Couldn't purge '%s' from Cloudflare. Unexpected JSON parse error.", url)
+                    logger.error("Couldn't purge from Cloudflare. Unexpected JSON parse error.")
```

Comment:
Would it be nice to include the list of URLs that failed for consistency here, to maintain current behavior? It'd be useful for logging purposes to know which URLs failed. Same comment applies to various other logging statements below.